docs/atlascli/command/atlas-config-rename.txt

```diff
@@ -53,3 +73,11 @@ Inherited Options
      - false
      - Profile to use from your configuration file.
 
+Examples
+--------
+
+.. code-block::
+
+   Rename a profile called myProfile to testProfile:
```

Comment:
**Question:** does this line need to be part of the code-block directive? It results in the description of the command being copied along with the command, which is annoying for a user that is trying to copy and paste the command only into the CLI. I see we did this in [other places](https://www.mongodb.com/docs/mongocli/stable/command/mongocli-auth-login/#examples) so maybe we had a reason for that, but just asking. I would think it goes above the code-block but below "Examples" for all of these.

Reply:
it doesn't but it's not a change for this PR, this was kind of discussed with @melissamahoney-mongodb when we worked on https://github.com/mongodb-labs/cobra2snooty, that lib is the one doing the formatting, we can look into how to improve this but it will require a change on the lib and probably fixing a lot of the current docs to adapt to it